cypress/support/e2e.ts

```diff
@@ -87,9 +87,10 @@ declare global {
       selectLGOption(label: string, option: string | RegExp): void;
       /**
        * Custom command to navigate to login page and login.
+       * @param username The username of the account to log in to.
```

Comment:
This may be a good place to describe the permission levels of each user since we can't document it well in the backend